lobsterpy/cli.py

```diff
@@ -555,20 +558,21 @@ def _element_basis(string: str):
 
 def _potcar_symbols(string: str):
     """
-    Parse string of potcar symbols and return a list
+    Parse string of potcar symbols and return a list.
+
     Args:
-        string:
+        string: string of potcar symbols
 
     Returns:
         list of potcar symbols
     """
-    potcar_symbols_list = string.split(" ")
+    return string.split(" ")
 
-    return potcar_symbols_list
+    # return potcar_symbols_list
```

Comment:
Can you remove this then?

Reply:
Yes, I will remove them, just left if there because I wanted to check if it does not break anything

Reply:
Done

Reply:
Done